apps/users/forms/registration.py

```diff
@@ -0,0 +1,110 @@
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Submit
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+
+from apps.users.models.user.constants import NATIONALITIES
+from apps.users.models.user.institution import InstitutionFontys, InstitutionTue
+from apps.users.models.user.profile import Profile
+
+
+class RegisterUserForm(UserCreationForm):
+
+    first_name = forms.CharField(max_length=30, required=True, initial="")
+    last_name = forms.CharField(max_length=30, required=True, initial="")
+    username = forms.EmailField(
+        max_length=254,
+        label="Institutional Email",
+        help_text="Required. Please fill in your institutional email address.",
+        required=True,
+        initial="",
+    )
+    email = forms.EmailField(
+        max_length=254,
+        label="Personal email",
+        required=False,
+        help_text="Optional, Please fill in a valid email address.",
+        initial="",
+    )
+    password1 = forms.CharField(widget=forms.PasswordInput, required=True, initial="")
+    password2 = forms.CharField(widget=forms.PasswordInput, required=True, initial="")
```

Comment:
set friendlier labels than "password1" and "password2".
"Password" and "Password(again)"/"Password confirmation"?